jobs/kpi-forecasting/Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN groupadd --gid ${USER_ID} ${GROUP_ID} && \
 
 WORKDIR ${HOME}
 
+RUN apt-get -y update
+RUN apt-get -y install git
 RUN pip install --upgrade pip
```

Comment:
For the pip install commands you should also include the `--no-cache-dir` flag. In general in the Dockerfile we try to keep each layer as small as possible.

Can read more about Dockerfile best practices here: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

Reply:
Thank you for the reference! I added `--no-cache-dir` to the pip commands.